src/notification/useNotification.jsx

```diff
@@ -1,6 +1,22 @@
 import { useContext } from "react";
 import { NotificationContext } from "./NotificationContext";
 
+/**
```

Comment:
write Readme.md copy from issue(https://github.com/InvincibleZeal/billbook/issues/57) and improve it.